recipes/datarobot-bp-workshop/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "datarobot-bp-workshop" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datarobot-bp-workshop-{{ version }}.tar.gz
+  sha256: 0f5e94206d89ea73d5798c88e145e3b2d8ebfadb258f30ca6374259813257582
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - datarobot >=2.27.0,<3
+    - python-graphviz >=0.16,<1
+    - networkx >=2.2
+    - python >=3.7
+
+test:
+  imports:
+    - datarobot_bp_workshop
+  commands:
+    - pip check
+
+about:
+  home: https://blueprint-workshop.datarobot.com
+  summary: This client library is designed to support building DataRobot blueprints.
+  license: Modified 3-Clause BSD
```

Comment:
I think it should maybe be:

```
license: BSD-3-Clause-Modification
```

or `BSD-3-Clause` but please consult https://spdx.org/licenses/ to be sure. You want to list a license from the "identifier" column.

Reply:
We have the extra line
> THIS SOFTWARE IS PROVIDED WITHOUT ANY LICENSE TO ANY PATENTS OR TRADEMARKS.
that I'm not seeing in the available licenses. Can I call this a proprietary license in this line or some other way to not use one of the existing options?

Reply:
Ok, maybe just change it to `Modified BSD-3-Clause`, but @conda-forge/staged-recipes does anyone want to chime in with a recommendation how to handle what is a `BSD-3-Clause` license with one additional line added to it ("THIS SOFTWARE IS PROVIDED WITHOUT ANY LICENSE TO ANY PATENTS OR TRADEMARKS.")?

Reply:
You can add an exception: https://spdx.org/licenses/exceptions-index.html

But if it's not in there I normally recommend going with _LicenseRef-BSD-3-Clause_ or something similar with _LicenseRef_.

Reply:
Thank you both for the help!!